hibike/README.md

```diff
@@ -430,12 +430,12 @@ Error handling
 
 ### Flashing
 
-Get a copy of arduino 1.8.1. This can be found in `PiE Team Root > Engineering > Hibike > Y10 > arduino-1.8.1-linux64.tar.xz`
+Get a copy of arduino 1.8.1. This can be found in `PiE Team Root > Engineering > Hibike > Y10 > arduino-1.8.1-linux64.tar.xz`. Alternately, visit the Arduino [webpage](https://www.arduino.cc/en/Main/OldSoftwareReleases) and grab a copy.
 
 Extract this on a GNU/Linux system using `tar -xvf arduino-1.8.1-linux64.tar.xz`. Move the entire folder to `/opt/`. You will probably have to `sudo` this.
 
 Then, run the setup script in hibike. `sudo ./setup.sh` in `piecentral/hibike` should do the trick. It will download a lot of stuff. You have effectively given up all your software freedoms at this point. Go cry a bit. Come back after asking Linus for forgiveness. 
 
-You're almost done, keep it up. Edit `/usr/bin/ard-reset-arduino`. You should sudo this. Don't worry, this is the last time you'll be violating your system. cahinge `ser.setBaudrate(1200)` to `se.baudrate = 1200`. Now your pour machine is ready to flash Arduinos (tm)
+You're almost done, keep it up. Edit `/usr/bin/ard-reset-arduino`. You should sudo this. Don't worry, this is the last time you'll be violating your system. Change `ser.setBaudrate(1200)` to `se.baudrate = 1200`. Now your poor machine is ready to flash Arduinos (tm)
 
-Flash by going to `piecentral/hibike`. The command to run is `./flash.sh SENSOR_NAME`. Figure out what the sensor names are I don't freaking know.   
+Flash by going to `piecentral/hibike`. The command to run is `./flash.sh SENSOR_NAME`. Figure out what the sensor names are in the Travis Makefile.
```

Comment:
Please just list the sensor names for ease of use:

The command to run is `./flash.sh SENSOR_NAME`, where `SENSOR_NAME` is one of 

- `BatteryBuzzer`
- `ExampleDevice`
- `LimitSwitch`
- `LineFollower`
- `Potentiometer`
- `RFID`
- `ServoControl`
- `TeamFlag`
- `YogiBear`